firefox-ios/Client/Frontend/Browser/BrowserViewController/Views/BrowserViewController.swift

```diff
@@ -498,7 +504,9 @@ class BrowserViewController: UIViewController,
                 dismissFakespotIfNeeded()
             }
 
+            // Update states for felt privacy
             updateInContentHomePanel(tabManager.selectedTab?.url)
+            setupMiddleButtonStatus(isLoading: false)
```

Comment:
update state for button - similar to how we need to update state for homepage
https://github.com/mozilla-mobile/firefox-ios/pull/17873